packages/flutter_tools/templates/app/linux.tmpl/Makefile

```diff
@@ -49,6 +49,7 @@ ALL_LIBS_OUT=$(FLUTTER_LIB_OUT) \
 # issues if they start with one or more '../'s.
 WRAPPER_ROOT=$(abspath $(FLUTTER_EPHEMERAL_DIR)/cpp_client_wrapper_glfw)
 WRAPPER_SOURCES= \
+	$(WRAPPER_ROOT)/flutter_engine.cc \
```

Comment:
Does this require a template # bump?

Reply:
Nope, this is only needed because the template is now using FlutterEngine; the refactoring in the engine is still only partial, so the view controller path is still separate from the engine path at the public API level.